07.model_serving.py

```
@@ -0,0 +1,162 @@
+        ),
+    ),
+)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Call the endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)